<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMenu</class>
 <widget class="QDialog" name="DialogMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>1071</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>971</width>
     <height>1021</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditEntrance">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>291</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Californian FB</family>
     <pointsize>14</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(213, 180, 143);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditDessert">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>710</y>
     <width>291</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Californian FB</family>
     <pointsize>14</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(213, 180, 143);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditMeal">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>390</y>
     <width>301</width>
     <height>591</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Californian FB</family>
     <pointsize>14</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(213, 180, 143);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
